spec/models/cohort_import_spec.rb

```diff
@@ -170,7 +167,12 @@
     let(:file) { "valid.csv" }
 
     context "when import_search_pds flag is enabled" do
-      before { Flipper.enable(:import_search_pds) }
+      before do
+        Flipper.enable(:import_search_pds)
+        allow(ProcessPatientChangesetJob).to receive(:perform_later)
+        allow(CommitPatientChangesetsJob).to receive(:perform_later)
```

Comment:
Does this make `expect(CommitPatientChangesetsJob).not_to have_been_enqueued` useless? As in, it couldn't possibly happen?

Maybe since this is a model spec it makes more sense to expect `PDSCascadingSearchJob.perform_later` to be called. Slightly more complicated with the `set(queue: :imports)` in there, but that's  ok.